testing-guidance/workflow-tests/scripting/script-python-tests.md

```diff
@@ -0,0 +1,41 @@
+# Python Scripts Workflow Tests
+
+Testing in this area should focus on the functionality of executing python scripts.
+
+## Common Issues to Watch For
+
+Test guidance will sometimes note specific issues to watch for. The common issues below should be watched for through all testing, even if unrelated to the current workflow being tested.
+1. Asset processor errors when importing/executing python scripts
+2. Warnings or errors that appear in the Editor console
+3. Unresponsive UI elements 
+
+### Platforms:
+
+- Windows
+- Linux
+
+### Documents and Common Terms
+
+[O3DE Documentation: Editor Python Bindings Gem](https://www.o3de.org/docs/user-guide/gems/reference/script/python/editor-python-bindings/)
+
+**Python Scripts**: Tool that lists python script samples and allows to execute those scripts.
+
+## Area: Opening Python Scripts tool and executing sample scripts
+
+### Project Requirements
+
+<ul>
+<li>Any project can be used, the Editor Python Bindings gem and its gem dependencies are required.</li>
+
+<li>White box Gem is required to visualize easily the script execution as it includes a python scripts that create various entities with meshes.</li>
+</ul>
```

Comment:
Use an MD list here since it is easier to read and is preferred for MD files when usable. HTML lists are only required inside of an MD table. It's a very unfortunate, and confusing, limitation of using an MD table.

Reply:
Applied MD list formatting for project requirements.